src/constants/Setting.js

```diff
@@ -0,0 +1,28 @@
+const WINNING_ARRAY = {
+  COUNTER: ['3개 일치', '4개 일치', '5개 일치', '5개 일치, 보너스 볼 일치', '6개 일치'],
+  AMOUNT: ['5,000', '50,000', '1,500,000', '30,000,000', '2,000,000,000'],
```

Comment:
```suggestion
  AMOUNT: [5000, 50_000, 1_500_000, 30_000_000, 2_000_000_000],
```
Number형으로 취급하는게 전체 금액 계산이나 수익률을 구할 때 더 좋을 것 같습니다! 출력할 때만 `amount.toLocaleString('ko-KR')` 이렇게 해주면 될 것 같아요.

Reply:
말씀대로 출력보다 계산에서 더 많이 사용을 해서 애초에 Number형으로 취급하는게 더 좋을 것 같네요 !!